src/containers/MapSelector.jsx

```diff
@@ -138,5 +138,5 @@ export default class SelectorData {
     return newCopy;
   }
 }
-SelectorData.GUIDED_MODE = 1;
-SelectorData.ADVANCED_MODE = 2;
+MapSelector.GUIDED_MODE = 1;
```

Comment:
Good point, done. `//The Map controls can be set to either Guided Mode (the default with UI widgets like checkboxes, etc) or Advanced Mode (which exposes full SQL capabilities).`